src/main/java/org/reaktivity/nukleus/mqtt/internal/stream/MqttServerFactory.java

```diff
@@ -1430,7 +1434,7 @@ private void doEncodeSuback(
             for (int i = 0; i < ackCount; i++)
             {
                 final int ackIndex = 1 << i;
-                subscriptions[i] = (byte) ((successMask & ackIndex) > 0 ? 0x00 : 0x8F);
+                subscriptions[i] = (byte) ((successMask & ackIndex) > 0 ? SUCCESS : TOPIC_FILTER_INVALID);
```

Comment:
Better to use `!= 0` instead of `> 0` for bitmask checks.
Cast to `byte` can be avoided if constants are defined as type `byte` instead.